src/embed/activity.ts

```
@@ -0,0 +1,409 @@
+
+  console.log('media', media);
+
+  if (!textOnly) {
+    if (media && media.length > 0) {
+      // @ts-expect-error doesn't know what to do with this
+      response['media_attachments'] = media.map(media => {
+        switch (media.type) {
+          case 'photo':
+            const image = media as APIPhoto;
+            return {
+              id: '114163769487684704',
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)